client/src/App.test.tsx

```diff
@@ -1,15 +1,177 @@
 import React from 'react'
-import { render, screen } from '@testing-library/react'
+import { screen, render } from '@testing-library/react'
+import { MemoryRouter } from 'react-router-dom'
 import App from './App'
-import * as utilities from './components/utilities'
+import { withMockFetch, renderWithRouter } from './components/testUtilities'
+import { IUserMeta } from './types'
+import { dummyBoards } from './components/DataEntry/_mocks'
+import {
+  // jaApiCalls,
+  aaApiCalls,
+} from './components/MultiJurisdictionAudit/_mocks'
+// import { manifestMocks } from './components/MultiJurisdictionAudit/useSetupMenuItems/_mocks'
 
 jest.unmock('react-toastify')
-jest.spyOn(utilities, 'api')
+
+const userMocks: { [key in 'ja' | 'aa']: IUserMeta } = {
+  ja: {
+    name: 'Han Solo',
+    email: 'falcon@gmail.com',
+    type: 'jurisdiction_admin',
+    organizations: [],
+    jurisdictions: [],
+  },
+  aa: {
+    name: 'Leia Organa',
+    email: 'princess@rebelalliance.com',
+    type: 'audit_admin',
+    organizations: [],
+    jurisdictions: [],
+  },
+}
```

Comment:
could you use `aaApiCalls` and `jaApiCalls` instead of these?

Reply:
@jonahkagan Good idea actually. I think I wrote these before I set up `aaApiCalls` and `jaApiCalls`, haha.